wurst/objediting/ObjectIds.wurst

```diff
@@ -4,7 +4,6 @@ import ErrorHandling
 import Wurstunit
 import Assets
 import LinkedList
-import StringUtils
```

Comment:
You can't just do breaking things like this without any replacement.

Reply:
Since its not public import, how can that break things? Also, the import was not used at all (maybe some functions which I actually moved from StringUtils to LinkedList were used by the ObjectIds, which was rendering this import obsolete)
Also, if I kept the import, there would be circular dependency problem.